src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -204,7 +205,11 @@ public void joystickDrive(double inputX, double inputY, double inputRotation) {
                     newInputY * m_maxVelocity,
                     newInputRotation * m_maxAngularVelocity);
         }
-        drive(chassisSpeeds);
+        if (Math.abs(newInputX) < 0.01 && Math.abs(newInputY) < 0.01 && Math.abs(newInputRotation) < 0.01){
+            stop();
+        } else {
```

Comment:
This test can go right after the slew limiters are applied, and then if you do that "stop()", just return immediately. That removes the need to the "else", and is a bit faster.